External/CoordGen/CMakeLists.txt

```diff
@@ -85,16 +85,16 @@ if(RDK_BUILD_COORDGEN_SUPPORT)
         CACHE STRING "CoordGen Include File" FORCE)
     file(GLOB CGSOURCES "${COORDGEN_DIR}/*.cpp")
     rdkit_library(coordgen ${CGSOURCES} SHARED)
-if(CMAKE_C_COMPILER_ID STREQUAL "Clang")
-    target_compile_options(coordgen PUBLIC -Wno-unused-but-set-variable)
-if(RDK_INSTALL_STATIC_LIBS)
-    target_compile_options(coordgen_static PUBLIC -Wno-unused-but-set-variable)
-endif()
-endif()
+    if(CMAKE_C_COMPILER_ID STREQUAL "Clang")
+      target_compile_options(coordgen PUBLIC -Wno-unused-but-set-variable)
+      if(RDK_INSTALL_STATIC_LIBS AND NOT RDK_BUILD_STATIC_LIBS_ONLY)
+        target_compile_options(coordgen_static PUBLIC -Wno-unused-but-set-variable)
+      endif()
+    endif()
     install(TARGETS coordgen DESTINATION ${RDKit_LibDir})
     set(RDK_COORDGEN_LIBS coordgen CACHE STRING "the external libraries" FORCE)
 
-  elseif(coordgen_FOUND)
+  elseif(COORDGEN_FORCE_BUILD OR(NOT coordgen_FOUND))
```

Comment:
I don't understand this change.
The goal of the original code here was to use a pre-existing coordgen library.

Reply:
Whoops, that was totally my mistake. I misread that as `else` and I _thought_ I was just removing a warning by making the `else` label consistent with the `if`, but that was an `elseif`, so I am effectively changing the condition. Later this evening I'll do a 30' session on dry chickpeas to expiate, unless you think I deserve a full hour.
I'll revert that.

Reply:
>  Later this evening I'll do a 30' session kneeling on dry chickpeas to expiate, unless you think I deserve a full hour. I'll revert that.

I think you did enough of that just to get this PR finished... no need this time!